spec.html

```diff
@@ -25194,39 +25210,43 @@ <h1>
         1. Set the running execution context's LexicalEnvironment to _env_.
         1. Let _instanceExtraInitializers_ be a new empty List.
         1. Let _staticExtraInitializers_ be a new empty List.
+        1. Let _metadataParent_ be ? Get(_constructorParent_, @@metadata).
+        1. If _metadataParent_ is *undefined*, let _metadataParent_ be %Object.prototype%.
```

Comment:
Should we bottom out in `null` instead of `%Object.prototype%`?

Reply:
Agreed, and updated, thanks for calling that out!